tasks/sf-sns-report/index.js

```diff
@@ -1,33 +1,14 @@
 'use strict';
 
-const get = require('lodash.get');
-const { publishReportSnsMessages } = require('@cumulus/api/lambdas/publish-reports');
 const cumulusMessageAdapter = require('@cumulus/cumulus-message-adapter-js');
 
-/**
- * Publishes ingest notifications based on the Cumulus execution message.
- *
- * @param  {Object} event - a Cumulus execution message that has been sent through the
- * Cumulus Message Adapter.
- * @returns {Promise<Object>} - Payload object from the Cumulus message
- */
-async function publishSnsMessage(event) {
-  const meta = get(event, 'input.meta', {});
-
-  const message = {
-    ...event.input,
-    meta: {
-      ...meta,
-      status: 'running'
-    }
-  };
-
-  await publishReportSnsMessages(message);
-
-  return get(message, 'payload', {});
-}
-
-exports.publishSnsMessage = publishSnsMessage;
+const throwDisabledError = () => {
+  throw new Error(
+    '@cumulus/sf-sns-report has been deprecated due to a change in reporting architecture. As the'
+    + ' reporting SNS topic and any consumers to listen to it have been removed, this task has'
```

Comment:
The topic hasn't been deprecated, has it? But pushing to the topic no longer results in records being saved/updated

Reply:
Hmm.. Arguably. I guess you are right in that a topic with the same name still exists, but it no longer serves the same purpose or have the same producers/consumers.

Reply:
OK. Your call if you want to update the language to only mention there are no consumers reporting output from the topics anymore.